docs/book/src/multiversion-tutorial/testdata/project/Makefile

```diff
@@ -38,9 +38,10 @@ help: ## Display this help.
 
 ##@ Development
 
+## crd:maxDescLen=0 - It will skip the description section while creating an annotation. This change will avoid annotations too long errors.
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
@laxmikantbpandhare I am OK with the workaround. Just a few nits:
a) Could we change the title so that makes it clear that it is about the docs/samples. Example:

:doc: fix multiversion tutorial sample error: metadata.annotations: Too long: must have at most 262144 bytes 

b) Could you please run `make manifests` in the /docs/book/src/multiversion-tutorial/testdata/project so that we can also merge the changes in the CRD? ( After we add this option it removes all descriptions )

c) Could we move the comment from to be on top of the line which has the `crd:maxDescLen=0 ` and use only one # since it is a comment and not a section? Therefore, wdyt about something like:

```sh
# Note that the option maxDescLen=0 was added in the default scaffold in order to sort out the issue
# Too long: must have at most 262144 bytes. By using kubectl apply to create / update resources an annotation 
# is created by K8s API to store the latest version of the resource ( kubectl.kubernetes.io/last-applied-configuration). 
# However, it has a size limit and if the CRD is too big with so many long descriptions as this one it will cause the failure. 
$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Reply:
@camilamacedo86 - I agree with you. I updated the PR.